Source/FieldSolver/SpectralSolver/SpectralHankelTransform/SpectralHankelTransformer.cpp

```diff
@@ -49,6 +49,7 @@ SpectralHankelTransformer::ExtractKrArray ()
             kr_array[ii] = kr_m_array[ir];
         });
     }
+    amrex::Gpu::synchronize();
```

Comment:
@WeiqunZhang can you quickly re-iterate why we need to synchronize here? Is the device memory synchronized to pinned memory at that place, too and `m_kr` then used on the host?